extensions/amp-story/1.0/amp-story-share.css

```diff
@@ -128,14 +128,9 @@
   background-position: 5px 5px !important;
 }
 
-.i-amphtml-story-share-icon:active,
 .i-amphtml-story-share-icon:focus {
-  outline: none !important;
-  box-shadow: none !important;
-}
-
-.i-amphtml-story-share-icon:active::before {
-  background-color: #787878 !important;
+  outline: 2px solid #0389ff !important;
```

Comment:
nit:
It's best practice to not override default browser state for a11y. I think we can remove this override. 
The outline below it is a nice touch though to help it be visible.

It might be worth putting a comment above this block stating that it's for a11y.

Reply:
The default for these elements (being custom elements / divs and not necessarily buttons) is to not have anything for the focus state, so this is super useful to know what the user is focusing on. I can add a comment, but I wonder if it's not clear enough if it's inside a rule with the `:focus` selector